manifests/7/7zip/7zip/21.07/7zip.7zip.installer.yaml

```diff
@@ -0,0 +1,82 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: 7zip.7zip
+PackageVersion: 21.07
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- 7z
+FileExtensions:
+- 7z
+- ar
+- arj
+- bzip2
+- cab
+- chm
+- cpio
+- cramfs
+- dmg
+- ext
+- fat
+- gpt
+- gzip
+- hfs
+- ihex
+- iso
+- lzh
+- lzma
+- mbr
+- msi
+- nsis
+- ntfs
+- qcow2
+- rar
+- rpm
+- squashfs
+- tar
+- udf
+- uefi
+- vdi
+- vhd
+- vmdk
+- wim
+- xar
+- xz
+- z
+- zip
+Installers:
+- Architecture: x64
+  InstallerType: exe

```

Comment:
```suggestion
  InstallerType: nullsoft
```

Reply:
The installer does not appear to have been created with Nullsoft Scriptable Install System (NSIS) ...but it does accept the same command line parameters, so I can't say No to your suggestion.  :-)

Reply:
Hmm, i see, since the 7zip installer accepts /S and /D there is no issues indeed, if you want you can go back to exe then add
```
InstallerSwitches:
  Silent: /S
  SilentWithProgress: /S
  InstallLocation: /D=<INSTALLPATH>
```
to the exe installers, wingetcreate will probably convert the InstallerType back to exe next time anways